### PR TITLE
Update PHP CS Fixer github action to use standard GA instead of prestashopcorp/github-action-php-cs-fixer

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | If we rely on `prestashopcorp/github-action-php-cs-fixer@master` the php version is frozen and this adds a layer of maintainance while the equivalent GitHub Action is not so big.<br/><br/>Also this repository should not depend on a PrestaShopCorp repository as it belongs to the OSS organization.
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| How to test?      | 
| Possible impacts? |

### CODE REVIEW

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -19,9 +19,20 @@ jobs:
...
+
+        -   name: Checkout
+            uses: actions/checkout@v2
+
+        -   name: Composer Install
+            run: composer install --ansi --prefer-dist --no-interaction --no-progress --quiet
 
-      - name: Run PHP-CS-Fixer
-        uses: prestashopcorp/github-action-php-cs-fixer@master
+        -   name: Run PHPCSFixer
```
Comment (Quetzacoalt91) [inline .github/workflows/php.yml:37]: Any reason to remove the spaces in PHP CS Fixer ?

Comment (matks) [reply .github/workflows/php.yml:37]: No, I can restore them ^^

File: .github/workflows/php.yml [hunk 2]
```diff
@@ -19,9 +19,20 @@ jobs:
   php-cs-fixer:
     name: PHP-CS-Fixer
     runs-on: ubuntu-latest
+    strategy:
+        matrix:
+            php: ['7.3', '7.4']
```
Comment (PierreRambaud) [inline .github/workflows/php.yml:24]:
```suggestion
      matrix:
        php: ['7.3', '7.4']
```

File: .github/workflows/php.yml [hunk 3]
```diff
@@ -19,9 +19,20 @@ jobs:
...
+        -   name: Setup PHP
+            uses: shivammathur/setup-php@v2
+            with:
+                php-version: ${{ matrix.php }}
+
+        -   name: Checkout
+            uses: actions/checkout@v2
+
+        -   name: Composer Install
+            run: composer install --ansi --prefer-dist --no-interaction --no-progress --quiet
```
Comment (PierreRambaud) [inline .github/workflows/php.yml:35]:
```suggestion
      - name: Setup PHP
        uses: shivammathur/setup-php@v2
        with:
          php-version: ${{ matrix.php }}

      - name: Checkout
        uses: actions/checkout@v2

      - name: Composer Install
        run: composer install --ansi --prefer-dist --no-interaction --no-progress --quiet
```

File: .github/workflows/php.yml [hunk 4]
```diff
@@ -19,9 +19,20 @@ jobs:
...
-        uses: actions/checkout@v2.0.0
+        uses: actions/checkout@v2
+
+      - name: Composer Install
+        run: composer install --ansi --prefer-dist --no-interaction --no-progress --quiet
 
-      - name: Run PHP-CS-Fixer
-        uses: prestashopcorp/github-action-php-cs-fixer@master
+        -   name: Run PHPCSFixer
+            run: ./vendor/bin/php-cs-fixer fix --dry-run --diff
```
Comment (PierreRambaud) [inline .github/workflows/php.yml:38]:
```suggestion
      - name: Run PHPCSFixer
        run: ./vendor/bin/php-cs-fixer fix --dry-run --diff
```

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank @matks I updated yoru PR to keep the two spaces indent

**Pull request merged**